### PR TITLE
feat(run): grpc-server-streaming sample

### PR DESCRIPTION
**Pull request by ahmetb**
Primarily forked and simplified run/grpc-ping.

Currently not associated to any tutorial but can be deployed easily from
instructions. It will be referred in a blog post. When there's a tutorial,
it can be used from there as well (by adding appropriate region tags).

Please **HOLD MERGING** until I give greenlight.

### CODE REVIEW

**Comment by ahmetb** [comment]
I'm seeing a failure from:

    license_test.go:71: run/grpc-server-streaming/pkg/api/v1/timeservice.pb.go: license header not present

this sounds a bit unreasonable since we're not supposed to edit autogenerated files. Similarly, other .pb.go files like https://github.com/GoogleCloudPlatform/golang-samples/blob/master/run/grpc-ping/pkg/api/v1/message.pb.go don't have a license header either?

**Comment by tbpg** [comment]
If the file shouldn't have the license header (please confirm), you can add an exception to https://github.com/GoogleCloudPlatform/golang-samples/blob/master/license_test.go#L42.

**Comment by ahmetb** [comment]
Thanks, I should've noticed that earlier. :)

**Comment by tbpg** [comment]
Closing this for now, but we can reopen when ready.

**Comment by ahmetb** [comment]
Reopening as the launch is current.
@grayside the wheels are in motion. It would be great if we can merge.

**Comment by tbpg** [comment]
In general, this repo is for samples that are included on cloud.google.com. If that's not in the plan, this would probably be a better fit for another repo.

**Comment by ahmetb** [comment]
Ideally, we’d have an accompanying tutorial in the long term that uses this sample.
However, for now we have a launch and we want to reference it from the launch blog.
Let me know how you want to proceed.

File: run/grpc-server-streaming/client/client.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
+// Copyright 2019 Google LLC
```
Comment (tbpg) [inline run/grpc-server-streaming/client/client.go:1]: 2020

File: run/grpc-server-streaming/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START run_grpc_dockerfile]
+
+# Use the offical golang image to create a binary.
+# This is based on Debian and sets the GOPATH to /go.
+# https://hub.docker.com/_/golang
+FROM golang:1.13-buster as builder
```
Comment (tbpg) [inline run/grpc-server-streaming/Dockerfile:20]: Consider 1.14.

**Pull request merged**